### ISSUE TITLE
train error: svd_cuda

### ISSUE THREAD

**Issue by viki-li** [andreas128/SRFlow#21]
Hello, thank you for your work.When I tried to train with my own datasets, I used the SRFlow_DF2K_4x model, but during the training process, "svd_cuda: the updating process of SBDSDC did not converage (error:11)" error appeared.Have you ever encountered the same reason in your training?Is there a solution?
![image](https://user-images.githubusercontent.com/77031692/108587338-0583b500-738e-11eb-8d83-620f8c27ae94.png)

**Comment by flybiubiu** [comment]
which cuda and pytorch version?

**Comment by viki-li** [comment]
Thank you for your response.&nbsp; I used CUDA 10. 0 and pytorch1.7. Are there any problems?&nbsp;

---Original---

**Comment by flybiubiu** [comment]
I train it with same problem before, But I run setup file all problem  are display.
If GPU is 3090,cuda version should be cuda 11.0

**Comment by andreas128** [comment]
Hi viki-li, this error usually occurs in the back propagation step of the 1x1 convolution, when the learning rate is too high.

Does it help to reduce the learning rate?
